core/Pos/Core/Genesis/Types.hs

```diff
@@ -1,52 +1,43 @@
+-- | Types related to genesis core data.
+
 module Pos.Core.Genesis.Types
        ( StakeDistribution (..)
-       , GenesisCoreData (..)
        , getTotalStake
+
+       , AddrDistribution
+       , GenesisCoreData (..)
+       , mkGenesisCoreData
+       , concatGenesisCoreData
        ) where
 
 import           Universum
 
-import           Data.Default       (Default (..))
-
-import           Pos.Core.Coin      (coinToInteger, sumCoins, unsafeAddCoin,
-                                     unsafeIntegerToCoin, unsafeMulCoin)
-import           Pos.Core.Constants (genesisKeysN)
-import           Pos.Core.Types     (Address, Coin, StakeholderId, mkCoin)
+import           Pos.Core.Coin  (coinToInteger, sumCoins, unsafeIntegerToCoin)
+import           Pos.Core.Types (Address, Coin, StakeholderId, mkCoin)
 
 -- | Stake distribution in genesis block.
--- FlatStakes is a flat distribution, i. e. each node has the same amount of coins.
--- BitcoinStakes is a Bitcoin mining pool-style ditribution.
 data StakeDistribution
```

Comment:
Do I understand correctly that this type represents not stakes but balances? If it's true, can you add a reference to CSL-1124 somewhere here?